ontology/clyh.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: ontology_detail
+id: clyh
+title: Clytia hemisphaerica Development and Anatomy Ontology
+build:
+  checkout: git clone https://github.com/EBISPOT/clyh_ontology.git
+  system: git
+  path: "."
+contact:
+  email: lucas.leclere@obs-vlfr.fr
+  label: Lucas Leclere
+  github: L-Leclere
+description: The Clytia hemisphaerica Development and Anatomy Ontology (CLYH) describes the anatomical and developmental features of the Clytia hemisphaerica life cycle.
+domain: anatomy
+homepage: https://github.com/EBISPOT/clyh_ontology
+products:
+  - id: clyh.owl
+  - id: clyh.obo
+dependencies:
+ - id: uberon
+ - id: ro
+ - id: iao
+tracker: https://github.com/EBISPOT/clyh_ontology/issues
+license:
+  url: http://creativecommons.org/licenses/by/3.0/
+  label: CC-BY
+activity_status: active
+---
+
+The Clytia hemisphaerica Development and Anatomy Ontology (CLYH) describes the anatomical and developmental features of the Clytia hemisphaerica life cycle. This species is a member of the phylum Cnidaria and belongs to the hydrozoan clade Leptothecata. Its life cycle comprises three stages: a planula larva, a benthic polyp colony, and a sexually-reproducing jellyfish (medu
```

Comment:
This paragraph ends in mid-sentence. Maybe it's just missing the word "medusa"?

Reply:
Your detail orientation is humbling. fixed. Thanks.